### DISCUSSION TITLE
Standup (Monday, November 7th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [BBGTC/food-bank-app#52] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Added custom colors to theme
 - Added basic navigation to signup

**What are your priorities for the next 24 hours?**:
 - Create priority list item
 - Polish theme

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for being a Django erudite

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Meeting with BB
 - Worked on #38 

**What are your priorities for the next 24 hours?**:
 - Work on front-end stuff
 - Send PR for #38 

**Blockers**:
 - None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Created db-models PR with @CarlosCRG19 
 - Started working on event card component

**What are your priorities for the next 24 hours?**:
 -  Finish card component
 - Work on Ken's assingment

**Blockers**:
 - None

**Shoutouts**:
 - 21 can you do something for me

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Redecided our plans for the rest of the sprint

**What are your priorities for the next 24 hours?**:
 - Assign tasks with the team
 - Start developing

**Blockers**:
 - We haven't assigned tasks yet

**Shoutouts**:
 - @BrandonMagana for a great PR and @CarlosCRG19 for the testing pipeline
